pysat/tests/test_meta.py

```diff
@@ -15,93 +15,113 @@
 class TestBasics():
     def setup(self):
         """Runs before every method to create a clean testing setup."""
-        self.meta = pysat.Meta()
         self.testInst = pysat.Instrument('pysat', 'testing',
                                          clean_level='clean')
+        self.meta = self.testInst.meta
+        self.dval = None
+        self.stime = [2009, 1]
+        self.out = None
+        self.default_name = ['long_name', 'axis', 'label']
+        self.default_nan = ['fill', 'value_min', 'value_max']
+        self.default_val = {'notes': '', 'units': '', 'desc': '',
+                            'scale': 'linear'}
 
     def teardown(self):
         """Runs after every method to clean up previous testing."""
-        del self.testInst
-        del self.meta
+        del self.testInst, self.meta, self.out, self.stime
+        del self.default_name, self.default_nan, self.default_val, self.dval
 
-    def test_meta_repr(self):
-        output = self.meta.__repr__()
-        assert isinstance(output, str)
-        assert output.find('pysat.MetaData') >= 0
+    def test_meta_settings(self):
+        """ Test the Meta settings for a specified value
+        """
+        if self.dval is None:
+            return
```

Comment:
If someone writes a new test that uses this function and forgets to set `self.dval`, this function will return and it will show up as a passed test.  Debugging this will be difficult since it will look like all lines are covered in coveralls since it is run for multiple tests.  Renaming the function to something that pytest won't gather up as a test (along with deleting lines 37 and 38) should fix this, and has the benefit of throwing an error should future tests forget to specify `self.dval`.
```suggestion
    def check_meta_settings(self):
        """ Test the Meta settings for a specified value
        """
```

Reply:
> Renaming the function to something that pytest won't gather up as a test (along with deleting lines 37 and 38) should fix this

How is this done?  Oh, through changing test to check?